### ISSUE TITLE
[ISSUE]: Not even opening when i click start_http.bat

### ISSUE THREAD

**Issue by nhigh1** [w-okada/voice-changer#1409]
### In a few words, describe your idea

123

### More information

Voice Changer Version
v.1.5.3.13

Operational System
windows 10 pro

GPU
1050 ti

Issue Description:
So i downloaded it and did everything right and extracted it to a different folder after making sure i downloaded the right thing and i click start_http and a window opens for a split second and closes and i can click it as much as i want but the same window just opens and closes in a split second !!!!!!!!! I want to say! It used to work for me, then I deleted it and now I'm trying to download it but it doesn't work.

**Comment by Kuuko-fokkusugaru** [comment]
Download the latest version v1.5.3.18a, your version is outdated.
Also download and install MS VS Resist:

https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist

**Comment by nhigh1** [comment]
does not work !

**Comment by nhigh1** [comment]
What else can I do?

**Comment by nhigh1** [comment]
![Снимок экрана (144)](https://github.com/user-attachments/assets/1fae79a8-0743-48c7-81eb-6958d5d1b25c)

**Comment by nhigh1** [comment]
it opens and closes for 1 second

**Comment by Kuuko-fokkusugaru** [comment]
Usually that happens to few people and it gets fixed after installing MS VS Redist.
Try the v2 instead of the old v1.

**Comment by idontwantausernamegoaway** [comment]
I also have this problem (i have the latest version installed) i try to open the bat file and it closes instantly

**Comment by idontwantausernamegoaway** [comment]
ok the problem fixed itself for a reason i dont even know

but now I cant hear myself in the thing so idk

**Comment by neru1023** [comment]
I have the same question.The screen flickers, and nothing can be opened.

**Comment by neru1023** [comment]
Could it be related to the Windows version?

**Comment by nhigh1** [comment]
hmm
yes

**Comment by Kuuko-fokkusugaru** [comment]
No, it have nothing to do with the Windows version. If you have the same issue, try the same solutions.

**Comment by neru1023** [comment]
Thanks, but it still doesn't work :(
